src/ControlSurface/Controls/Position/ScorePositionPoint.qml

```diff
@@ -15,17 +15,18 @@ Rectangle {
     property string controlUuid
     property string controlSurfacePath
 
-    property real controlX: vertical.x * (controlDomain / position.height)
-    property real controlY: horizontal.y * (controlDomain / position.height)
+    property real controlX: controlMin + vertical.x * (controlDomain / position.height)
+    property real controlY: controlMin + horizontal.y * (controlDomain / position.height)
```

Comment:
same, in general when there's a min there's also a max ? but maybe there's a good reason